client/package.json

```diff
@@ -78,12 +79,12 @@
     "html-lint": "htmlhint -c .htmlhintrc src",
     "lint": "npm run js-lint && npm run css-lint && npm run html-lint",
     "coverage": "npm run test && nyc report --reporter=lcov --reporter=text",
-    "start": "env NODE_OPTIONS=--max_old_space_size=3584 gulp",
-    "build": "env NODE_OPTIONS=--max_old_space_size=3584 gulp build",
-    "build:prod": "env NODE_ENV=production NODE_OPTIONS=--max_old_space_size=3584 gulp build",
+    "start": "cross-env NODE_OPTIONS=--max_old_space_size=3584 gulp",
```

Comment:
Nice to have cross-env.
I was ignoring Windows because I didn't expect people to play this project...